ThunderDome/ThunderDome/monsterBlock.cs

```diff
@@ -38,9 +111,9 @@ public monsterBlock()
 
 
         //Display healthbar
-        public static string healthBar(monsterBlock monster)
+        public string healthBar()
         {
-            double doubleCurrentHealth = ((double)monster.currentHealth / (double)monster.MaxHealth) * 10;
+            double doubleCurrentHealth = ((double)this.currentHealth / (double)this.MaxHealth) * 10;
```

Comment:
notice the use of `this`. You should cover this in class, if not ask.  But instead of passing the object into the method you simply refer to the current instance of the object as `this`